src/components/Canary/CanaryPopup/CheckTitle.tsx

```diff
@@ -4,7 +4,18 @@ import { usePrevious } from "../../../utils/hooks";
 import { Badge } from "../../Badge";
 import { Icon } from "../../Icon";
 
-export function CheckTitle({ check, className, size = "large", ...rest }) {
+type Props = {
+  check: any;
+  className?: string;
```

Comment:
className prop is already there in the type `React.HTMLAttributes<HTMLDivElement>`, no need to add it here.

Reply:
Yeah, that makes absolute sense.